docs/contribute.rst

```diff
@@ -183,6 +183,22 @@ Here's the overview of core classes::
     └── Run
 
 
+Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Object hierarchy is following: ``Run`` -> ``Plans`` -> ``Steps``
+-> ``Plugins``, where the ``Run`` is on the top of this hierarchy.
+The objects have the ``parent`` attribute, that is pointing to
+the parent in which current class is contained.
+
+The ``node`` attribute of ``Test``, ``Plan`` and ``Story``
+instances references the original fmf metadata node from which
```

Comment:
not sure if I understand this. `instances references the original fmf metadata node` - from my perspective my understanding is that node contains same data as `Tree.tree`.
Maybe some different sentence like: `instances copies FMF data from original metadata node` 

if I understand it well @psss . :-)

Reply:
The `node` references the corresponding leaf of the fmf metadata tree from which test/plan/story has been created. It does not copy any information. Maybe using `leaf` would make it a bit more clear?

Reply:
What about this:

> The `node` attribute ... references the original leaf node of the fmf metadata tree from which the respective test/plan/story has been created.

Does this sound more clear?

Reply:
yes, it is more clear now.

Reply:
Thanks, updated.